### PR TITLE
Pandas: Fixed typo

### LINKED ISSUE #4382: Pandas Cheat Sheet: Typo in First Aggregate Example
When searching for ["pandas cheatsheet"](https://duckduckgo.com/?q=pandas+cheatsheet&t=osx&ia=cheatsheet&iax=1) there is a typo. As the screenshot shows, 

    df.groupby('column1'.sum())

is missing a `)` and should be fixed to:

    df.groupby('column1').sum())

screen shot 2017-07-25 at 4 23 35 pm

------
IA Page:  http://duck.co/ia/view/pandas_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Hitechverma

### PR DESCRIPTION
**Pull request by CedArctic**
There was a trailing ")" on line 95 as reported in issue #4382.

## Related Issues and Discussions

**Fixes #4382.** Fixed a typo: There was a trailing ")" in line 95. 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/pandas_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
@CedArctic welcome! 👋

**Comment by CedArctic** [comment]
@pjhampton Thanks for the welcoming, looking forward to contributing more in the future :)

_Sent from my Oneplus A0001 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: share/goodie/cheat_sheets/json/pandas.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
         }],
         "Aggregate": [{
             "val": "Spliting the data into groups and apply fucntions to each groups indivdually",
-            "key": "[df.groupby('column1'.sum())] [df.groupby(\\['column1','column2'\\]).count()]"
+            "key": "[df.groupby('column1'.sum()] [df.groupby(\\['column1','column2'\\]).count()]"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/pandas.json]: @CedArctic the correct fix for this should be `df.groupby('column1').sum()`

Comment (CedArctic) [inline share/goodie/cheat_sheets/json/pandas.json]: @sahildua2305 You're right, must have overlooked that. I edited the commit accordingly. Thank you :)

**Pull request merged**